### DISCUSSION TITLE
Access the state instance of a module from the JS file

### DISCUSSION THREAD

**Discussion by 72fcosta** [quasarframework/quasar#10288] [General - Components / Directives / etc]
Hello, greetings.

I am trying to access the state instance of a module from the JS file, however when importing the store:
`import store from './../store'`

 I get the function:
`      ƒ ()
      /* { ssrContext } */
      {
      const Store = new vuex__WEBPACK_IMPORTED_MODULE_1__["default"].Store({
         modules: {
            global: _global__WEBPACK_IMPORTED_MODULE_2__["default"],
            ...`

- quasar: 1.15.22
- Modules generated by quasar new store <store_name>
- index.js of store folder:
`      import Vue from 'vue'
      import Vuex from 'vuex'

      import global from './global'
      import order from './order'
      import panel from './panel'

      Vue.use(Vuex)

      export default function (/* { ssrContext } */) {
         const Store = new Vuex.Store({
            modules: {
               global,
               order,
               panel
            },
            strict: process.env.DEBUGGING
         })
         return Store
      }`

Can you help me?

**Comment by hawkeye64** [comment]
@72fcosta Almost there. Just needs a bit of tweaking...

```js
import global from './global'
import order from './order'
import panel from './panel'

Vue.use(Vuex)

let store
export default function (/* { ssrContext } */) {
   const Store = new Vuex.Store({
      modules: {
         global,
         order,
         panel
      },
      strict: process.env.DEBUGGING
   })
  store = Store
   return Store
}

export store
```
Now, from any file, you can:
```js
import { store } from '../path/to/store/file'
```

**Reply by 72fcosta** [reply]
Thanks Jeff.
I do not know what I did wrong this time.
![image](https://user-images.githubusercontent.com/25357754/128259199-d46b0e17-9ccc-4aa8-8633-275f6e3cd129.png)

**Reply by hawkeye64** [reply]
Do it as `export { store }` then

**Reply by 72fcosta** [reply]
Jeff I have always been able to use medianally quasar / vue / js ... except for import / export. Where do I get 10% of your knowledge for this type of problem? Thanks Thanks

**Reply by hawkeye64** [reply]
The more you use it, the better it gets. However, I found if you help and mentor others, that forces you to learn faster, just to stay that single baby step ahead of others. :)
